### PR TITLE
Add a draft agenda for TPAC

### PR DESCRIPTION
**Pull request by martinthomson**
We're meeting later this week, but I thought that I would get something
in place now.

I'm not certain about the feasibility of the remote attendance setup
using Zoom as I have here, but until we work out details, this will have
to do.

### CODE REVIEW

File: 2022/tpac/agenda.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Agenda
+
+* Setup, Welcome, Code of Conduct (chairs; 10m)
+* FedCM CG Coordination (Heather Flanagan: 30m)
+  * Updates on FedCM work
+  * Discussion of privacy-related topics
+  * Decision tree flows
+* Overview of group, brief status of open work items and proposals (chairs; 20m)
+* Storage Access (Johann ?; 40m)
```
Comment (johannhof) [inline 2022/tpac/agenda.md:33]: Happy to present something, won't be super polished but I can give a quick intro.

File: 2022/tpac/agenda.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+fee](https://www.w3.org/2022/09/TPAC/registration.html#fees), though the W3C
+offers a [fee waiver](https://www.w3.org/2022/09/TPAC/registration.html#waiver)
+for those who might find a fee to be a barrier to participation.
+
+[W3C Privacy CG Calendar](https://www.w3.org/groups/cg/privacycg/calendar)
+
+## Agenda
+
+* Setup, Welcome, Code of Conduct (chairs; 10m)
+* FedCM CG Coordination (Heather Flanagan: 30m)
```
Comment (samuelgoto) [inline 2022/tpac/agenda.md:28]:
```suggestion
* FedID CG Coordination (Heather Flanagan: 30m)
```

**Comment by wanderview** [comment]
Chrome has a new explainer around bounce tracking mitigations we'd like to discuss at TPAC:

https://github.com/wanderview/bounce-tracking-mitigations/blob/main/explainer.md

As a side note, does privacy CG really only have 2 hours at TPAC?  I expected this to be the more typical day-long session.

File: 2022/tpac/agenda.md (hunk 1 above)
Comment (martinthomson) [reply 2022/tpac/agenda.md:33]: Thanks.  We're hoping to develop a plan to graduate this work, so focus on those open issues you think need to be more urgently resolved and where you think this work needs to go.

**Comment by martinthomson** [comment]
@wanderview,

> does privacy CG really only have 2 hours at TPAC? 

If everyone asked for that much time, we'd be guaranteed conflicts.  With these 2h meetings at least all the different groups are able to spend some time and have some hope of attracting the sort of broader participation that an in-person meeting is best for.  A bunch of all-day meetings might work, but you wouldn't be able to ensure that you would get focused attention for important stuff.  For instance, web apps looks like it is going to be a room that hosts a bunch of different meetings with different people, which makes it hard for those of us with a pile of conflicts to justify spending any time with them.

As for your request, it's a little bit of short notice.  As you can see, we've already allocated time to work items.  So if we have time remaining, we'll let you do a pitch for this.  I'm sure that the folks on [the navigation tracking work](https://privacycg.github.io/nav-tracking-mitigations/) would be interested in your input.

**Comment by wanderview** [comment]
> If everyone asked for that much time, we'd be guaranteed conflicts.

Every TPAC I've attended WGs had full day sessions with plenty of time for agenda.  2 hours is a bit surprising to me.

> little bit of short notice

You opened this PR 3 days ago.  Monday was a holiday in the US.  I added my request the first working day after this PR was created.  I'm not sure what you expected for gathering input from others.

Anyway, I guess we'll make do.  Thanks.

**Comment by hober** [comment]
> > If everyone asked for that much time, we'd be guaranteed conflicts.
> 
> Every TPAC I've attended WGs had full day sessions with plenty of time for agenda. 2 hours is a bit surprising to me.

Alas, we're not a WG.

**Pull request merged**